content/tutorials/content/single-page-tutorials/docusaurus.config.js

```diff
@@ -13,6 +13,13 @@ module.exports = {
         route: 'send-iota-tokens-with-javascript',
         tags: ['text'],
       },
+      {
+        title: 'Run a Wasp Node',
+        description: 'In this tutorial you will learn how to run a wasp node.',
+        preview: 'IOTA-Smart-Contract-Tutorials-A.jpg',
+        route: 'https://wiki.iota.org/smart-contracts/guide/chains_and_nodes/running-a-node',
```

Comment:
```suggestion
        route: 'smart-contracts/guide/chains_and_nodes/running-a-node',
```

Please don't use a domain. Only the path is enough. Otherwise test build would break 👍